lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx-simulators/string/main.cpp

```diff
@@ -77,7 +89,12 @@ template <class _CharT, class _Traits, class _Allocator> class basic_string {
     };
 #else // !SUBCLASS_PADDING
 
+#ifdef UB_PADDING
     unsigned char __padding[sizeof(value_type) - 1];
```

Comment:
Was that (the zero sized array) really UB?  Or just a compiler extension?

Reply:
Good point! I'll rename it to `NON_STANDARD_PADDING`